asr-python-advanced-tao-ngram-pretrain.ipynb

```diff
@@ -157,20 +157,20 @@
     "---\n",
     "<a id='isc-prepare-data'></a>\n",
     "### Preparing the dataset\n",
-    "#### Librispeech LM Normalized dataset\n",
-    "For this tutorial, we use the normalized version of the LibriSpeech LM dataset to train our N-gram language model. The normalized version of the LibriSpeech LM dataset is available [here](https://www.openslr.org/11/).\n",
+    "#### Librispeech normalized text dataset\n",
+    "For this tutorial, we use the normalized version of the LibriSpeech text dataset to train our n-gram language model. The normalized version of the LibriSpeech text dataset is available [here](https://www.openslr.org/11/).\n",
     "\n",
-    "#### LibriSpeech dev-clean dataset\n",
-    "For this tutorial, we also use the clean version of the LibriSpeech development set to evaluate our N-gram language model. The clean version of the LibriSpeech development set is available [here](https://www.openslr.org/12/)."
+    "#### LibriSpeech dev-clean speech dataset\n",
+    "For this tutorial, we also use the clean version of the LibriSpeech speech dataset to evaluate our n-gram language model. The clean version of the LibriSpeech speech dataset is available [here](https://www.openslr.org/12/)."
    ]
   },
   {
    "cell_type": "markdown",
    "metadata": {},
    "source": [
     "### Downloading the dataset\n",
```

Comment:
Instead of having multiple short sections with very similar headings, can we just combine them?  Have a single section called "Preparing and Downloading the Dataset"?

Reply:
No problem. I combine this two section and pushed the changes.